src/TestRunner.js

```diff
@@ -11,10 +11,11 @@
 //              tests.
 export default class TestRunner {
 
-  constructor(component, testSuites, startDelay, sendReport) {
+  constructor(component, testSuites, startDelay, reporter, sendReport) {
     this.component = component;
     this.testSuites = testSuites;
     this.startDelay = startDelay;
+    this.sendReport = reporter;
```

Comment:
`this.sendReport` vs `sendReport` is a little confusing; this looks like a typo at first. Could this just be `this.reporter`?